src/repo/learnocaml_exercise.ml

```diff
@@ -292,7 +319,10 @@ module File = struct
           read_file prepare ;
           read_file solution ;
           read_file test ;
+          read_file depend ;
           (* read_max_score () *) ] >>= fun () ->
+      join (let dep = try get depend !ex with Missing_file _ -> "" in 
```

Comment:
I am always suspicious when I see an exception being ignored like this. Can you justify this? Shouldn't you at least inform the user? or even stop with an error?